.github/workflows/deploy.yml

```
@@ -0,0 +1,21 @@
+name: Deploy BlueOS Extension Image
+
+on:
+  push:
+  pull_request:
+  # Run manually
+  workflow_dispatch:
+  # NOTE: caches may be removed if not run weekly
+  #  -> may be worth scheduling for every 6 days
+
+jobs:
+  deploy-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deploy BlueOS Extension
+        uses: BlueOS-community/Deploy-BlueOS-Extension@v1
```

```suggestion
        uses: BlueOS-community/Deploy-BlueOS-Extension@v1.0.1
```

We made a small update recently so it doesn't run as long when there aren't credentials provided - may as well use the latest :-)